Automattic/jetpack
# Tests: Changing factory method calls in tests

#### Changes proposed in this Pull Request:

* This PR changes all instances of `$this->factory` with `self::factory()`, that occur within Jetpack test files. 
* WordPress Core [removed magic methods here](https://github.com/WordPress/wordpress-develop/commit/12f501271829ec35f03d5f7fe5643204c38cb400), which in turn led to [a couple of our tests breaking](https://github.com/Automattic/jetpack/pull/26084) yesterday. [This was their cleanup commit](https://github.com/WordPress/wordpress-develop/commit/ff56396bf36cfc9ff8ebef54642a95775f5392aa) for reference.
* This PR cleans up all other tests using the same call, even though they're not causing any problems at this stage.

#### Other information:


#### Jetpack product discussion

p1662549402775149/1662548337.184269-slack-C03QNBQKG73

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* This shouldn't need to be tested locally as the tests that run in this PR via actions should ideally pass, however you can also run Jetpack PHP tests (for the Jetpack plugin) locally too.
